Hippo.Core/Domain/Order.cs

```diff
@@ -0,0 +1,104 @@
+﻿using Microsoft.EntityFrameworkCore;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Text;
+using System.Text.Json.Serialization;
+using System.Threading.Tasks;
+
+namespace Hippo.Core.Domain
+{
+    public class Order
+    {
+        [Key]
+        public int Id { get; set; }
+
+        [Required]
+        public string Category { get; set; }
+        [Required]
+        [MaxLength(50)]
+        public string Name { get; set; }
+        [MaxLength(250)]
+        public string Description { get; set; }
+        [MaxLength(150)]
+        public string ExternalReference { get; set; }
+        [Required]
+        [Range(0.01, double.MaxValue)]
+        public decimal Price { get; set; }
```

Comment:
we'll need "units" and then probably change this to "UnitPrice".  Units would be like "TB" for a terabyte of storage, "Point" for a fair share point, etc.  It's just for display purposes and can default to something generic, but I think we'll want it